docs/config_search.md

```diff
@@ -19,6 +19,10 @@ limitations under the License.
 Model Analyzer's `profile` subcommand supports **automatic** and **manual**
```

Comment:
Should we differentiate between brute and quick at the top level? And then break brute down into automatic vs manual?

Reply:
I don't think we need to. Brute is our default, so the user doesn't need to explicitly specifying, therefore I don't think we need to call this out, as our examples work correctly (now if quick becomes our default, then that's a different story...)

Reply:
This first paragraph should introduce the different modes and provide links to the more detailed sections below. 

At a high level, we have Brute and Quick.  Brute has subcategories automatic brute and manual brute. I think the sections should be renamed as such.

Reply:
Done